model/clusters_mgmt/v1/azure_node_pool_type.model

```diff
@@ -75,4 +75,10 @@ struct AzureNodePool {
     // If not specified, Server-Side Encryption (SSE) on the OS Disks of the Nodes of the Node Pool 
     // is performed with platform managed keys.
     OsDiskSseEncryptionSetResourceId String
+
+    // OsDisk contains the configuration for the OS disk used by the nodes in the Node Pool.
+    // This includes settings such as disk size, storage account type, persistence mode (Persistent or Ephemeral),
```

Comment:
I don't think we need the part:
```
// This includes settings such as disk size, storage account type, persistence mode (Persistent or Ephemeral),
Comment
// and optional server-side encryption using a customer-managed Disk Encryption Set (DES).
```

that can be documented in the corresponding inner attributes.

Same for 
```
    // If not specified, a 30 GiB Persistent OS disk with Premium_LRS is used.
``